web/src/frontend/src/nav-soknad/faktum/TextareaFaktum.tsx

```diff
@@ -69,6 +69,7 @@ class TextareaFaktum extends React.Component<Props, {}> {
 				? this.props.getPropertyVerdi()
 				: this.props.getFaktumVerdi()) || "";
 		return (
+			<div title="Max 50">
```

Comment:
Hva er et her?

Reply:
...ledetekst som ikke ligger soknadsosialhjelp-tekster. We fix.